check_diff/tests/git.rs

```diff
@@ -1,33 +1,25 @@
 use check_diff::clone_git_repo;
 
 use std::fs;
-use tempfile::{tempdir, Builder};
-use tracing::error;
+use tempfile::Builder;
 
 #[test]
 fn clone_repo_test() {
-    let dir = tempdir().unwrap();
-    match dir {
-        Ok(d) => {
-            let sample_repo = "https://github.com/rust-lang/rustfmt.git";
-            let dest_path = d.path();
-            let result = clone_git_repo(sample_repo, dest_path);
-            assert_eq!(result.is_ok(), true);
-            let directory = fs::read_dir(dest_path);
-            // check that the directory is non-empty
-            assert_eq!(directory.iter().next().is_none(), false);
-            // check whether a .git folder exists in directory
-            let mut is_git_present = false;
-            for path in directory.unwrap() {
-                if path.unwrap().file_name() == ".git" {
-                    is_git_present = true;
-                }
-            }
-            assert_eq!(is_git_present, true);
-        }
-        Err(e) => {
-            error!("Error from building: {}", e);
-            assert_eq!(1, 2);
+    // Creates an empty directory in the current working directory
+    let dir = Builder::new().tempdir_in("").unwrap();
+    let sample_repo = "https://github.com/rust-lang/rustfmt.git";
+    let dest_path = dir.path();
+    let result = clone_git_repo(sample_repo, dest_path);
+    assert_eq!(result.is_ok(), true);
```

Comment:
Simpler to use `assert!` here since you're testing a boolean value.
```suggestion
    assert!(result.is_ok());
```